### PR TITLE
feat: links page

### PR DESCRIPTION
**Pull request by csosamorales**
## Why?

Clear and short explanation here.

## How?

- Done A (replace with a breakdown of the steps)
- Done B
- Done C

## Tickets?

- [Ticket 1](the-ticket-url-here)
- [Ticket 2](the-ticket-url-here)
- [Ticket 3](the-ticket-url-here)

## Contribution checklist?

- [ ] The commit messages are detailed
- [ ] The `build` command runs locally
- [ ] Assets or static content are linked and stored in the project
- [ ] Document filename is named after the slug
- [ ] You've reviewed spelling using a grammar checker
- [ ] For documentation, guides or references, you've tested the commands and steps
- [ ] You've done enough research before writing

## Security checklist?

- [ ] Sensitive data has been identified and is being protected properly
- [ ] Injection has been prevented (parameterized queries, no eval or system calls)
- [ ] The Components are escaping output (to prevent XSS)

## References?

Optionally, provide references such as links

## Preview?

Optionally, provide the preview url here

### CODE REVIEW

**Review by bcamarneiro — DISMISSED** [review]

File: src/components/Links/index.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import { LinkButton } from './LinkButton';
+import type { CollectionEntry } from 'astro:content';
+import Link from '@components/Link';
+
+interface LinksProps {
+  featuredPost: CollectionEntry<'blog'>;
+}
+
+const Links: React.FC<LinksProps> = ({ featuredPost }) => {
+  const newsletterFeatureFlag = false;
```
Comment (bcamarneiro) [inline src/components/Links/index.tsx:17]: This is to be disabled for now as there are some decisions yet being made on the background regarding the usage of beehiiv

File: src/components/BiweeklySubscription.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        return;
+      }
+
+      throw new Error('Missing environment variables');
+    } catch (error) {
+      console.log('Request failed', error);
+    } finally {
+      setIsSubscribing(false);
+    }
+  };
```
Comment (heldrida) [inline src/components/BiweeklySubscription.tsx:64]: Removed redundancy given url and apiKey throws earlier. Also, removed a console.log.

```suggestion
const handleSubscription = async () => {
  setIsSubscribing(true);
  try {
    const response = await fetch(url, {
      method: 'POST',
      headers: {
        Authorization: `Bearer ${apiKey}`,
        'Content-Type': 'application/json',
      },
      body: JSON.stringify({
        email,
      }),
    });
  } catch (error) {
    console.log('Request failed', error);
  } finally {
    setIsSubscribing(false);
  }
};

```

File: src/pages/links.astro [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+import Layout from '@layouts/Page.astro';
+import settings from '@base/settings.json';
+import LinksModule from '@components/Links';
+import { getCollection } from 'astro:content';
+import type { CollectionEntry } from 'astro:content';
+
+// To change the featured post, replace the target post slug
```
Comment (heldrida) [inline src/pages/links.astro:8]: Replace implementation to use settings.json.

**Review by heldrida — COMMENTED** [review]
Let’s do our best to summarize the PR's by completing the form, making it easier for our teammates to switch context and provide meaningful feedback. Thanks!

File: README.md [hunk 4]
```diff
@@ -237,6 +239,14 @@ In the world where text is text, I show you an image:
 
 💡 Would like to place static video content? Learn how to work with video content [here](#video-content).
 
+### 🚩New blog image for links page
```
[the remark this answers is not in the document]
Comment (bcamarneiro) [reply README.md:242]: fixed

File: src/pages/links.astro (hunk 3 above)
Comment (bcamarneiro) [reply src/pages/links.astro]: done

**Review by angarita-dev — DISMISSED** [review]

**Comment by angarita-dev** [comment]
Approved @bcamarneiro I would say lets just check if the spell checker is breaking with this changes or if there's any action required there

File: src/components/Links/index.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+      <LinkButton
+        href={settings.site.production.url}
+        ariaLabel="Fleek website"
+        label="Website"
+      />
+      <LinkButton href="/blog/" ariaLabel="Fleek Blog" label="Blog" />
+      <LinkButton href="/docs/" ariaLabel="Fleek docs" label="Documentation" />
+      <Button variant="primary-ghost" className="w-full" size="sm">
+        <a
+          aria-label="Fleek X/Twitter account"
```
Comment (csosamorales) [inline src/components/Links/index.tsx:69]:
```suggestion
          aria-label="Fleek Platform"
```

**Review by bcamarneiro — APPROVED** [review]

**Pull request merged**